blog/posts/making-the-remix-austin-blog.mdx

```diff
@@ -0,0 +1,214 @@
+to this problem. These solutions work, and Colby's solution, inspired by [this Remix stack](https://github.com/freekrai/remix-blog),
+is hosted on [his Github](https://github.com/colbywhite/colbywhite.dev).
+
+<BlogImage
+  src={colbyWhite}
+  alt="Colby White's website"
+  caption="Pay Colby's website a visit if you can"
+/>
+
+So my next move was simple? Steal his work!
+
+With credit, of course! But hey, don't repeat work that's already been done for you. And thanks
+almost exclusively to my timing in this project, I was the unlikely benefactor of Colby's
+hard work <span className="text-xxs">Hehehehe</span>. I branched off a [previous PR for an initial blog implementation from Seth Davis](https://github.com/remix-austin/remixaustin-com/pull/86)
+and then got to work. I explored the Github repo for Colby's website, found the pieces concerned
+with parsing `.mdx` files into a consumable format, and then whole-hog copied and
+pasted that stuff over.
+
+I gotta admit, during the whole process, I only had about a 50% understanding of how everything worked.
+I just kind of let the compiler act as my guide. Module not found error?
+Great! What did I not steal that I was supposed to? Let me go back to Colby's
+repo and find out.
+
+With about two hours of copy and paste, some compiler errors, and a few mock blog files, I was
+able to get an [initial working solution in our site](https://github.com/remix-austin/remixaustin-com/pull/91/commits/971f180faa14c8dc672d8cb0ee66866ced24a248).
+
+## We got the goods. Now what?
+
+The solution technically worked, but it was lacking quite a bit. The first being understanding.
+I'm no fan of "copy and paste" into production code. Sure, publicly available solutions are great
+starting points. But you shouldn't put any code into a codebase that you don't understand.
+
+I took a good long look at what I copied and got my head around how it worked. The basic idea was this:
+there's a NodeJS script that takes all of the `.mdx` files and any of their dependencies
+and compiles them using `mdx-bundler`. The final result is a JSON array, with each post as an
+object in that array. Those objects have some frontmatter and a raw source string for a React
+component. That raw React component is the full content of the blog post.
+
+The array then gets written into a file called `post-cache.json` in our `/public`
+directory, making it available to all loaders in the site. Loaders for the blog list route and the
+individual blog post route make a fetch for the file, pick out the data they need, and pass the
+raw React component to a helper function from `mdx-bundler` which converts into a regular
+React component, and finally we display it in our page like good ol' JSX.
+
+And lastly, all `.mdx` files are watched by Node when we run this script in development mode. If any
+of their content changes, `post-cache.json` is regenerated with the updated contents.
+
+<BlogImage
+  src={mattConfused}
+  alt="Not confused at all"
```

Comment:
While I think this alt tag is funny, unfortunately I don't think a non-sighted user would quite understand it 😕. Mind if we update it to something a little more descriptive?

Reply:
Fixed it